lib/licensed/sources/npm.rb

```diff
@@ -135,6 +135,14 @@ def yarn_lock_present
       def include_non_production?
         config.dig("npm", "production_only") == false
       end
+
+      def missing_peer?(parent, dependency, name)
+        dependency["peerMissing"] || dependency["missing"] && peer_dependency?(parent, name)
```

Comment:
I _think_ you'd want parentheses around the last two checks right?  `peer_dependency?` would only be called as additional validation on `dependency["missing"]`?
```suggestion
        dependency["peerMissing"] || (dependency["missing"] && peer_dependency?(parent, name))
```